### PR TITLE
update label_image.py

### PR DESCRIPTION
**Pull request by freedomtan**
1. add build rule for label_image.py
2. remove extraneous semicolons

### CODE REVIEW

**Comment by sb2nov** [comment]
@gunan the buildifier error seems to be suggesting a noop. What does that mean?

File: tensorflow/examples/label_image/BUILD [hunk 1]
```diff
@@ -51,6 +51,16 @@ tf_cc_binary(
     }),
 )
 
+py_binary(
+    name = "label_image_py",
+    main = "label_image.py",
```
Comment (gunan) [inline tensorflow/examples/label_image/BUILD]: main should come after srcs.

Comment (sb2nov) [reply tensorflow/examples/label_image/BUILD]: Ahh interesting. @freedomtan can you change that.

Comment (freedomtan) [reply tensorflow/examples/label_image/BUILD]: @sb2nov done.

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
@gunan good to go if build passes?

**Review by sb2nov — APPROVED** [review]

**Comment by drpngx** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
@caisq looks like there are some failures on master, right?

**Comment by caisq** [comment]
Some of the failures do not seem to be existing on master. For example this one:
http://ci.tensorflow.org/job/tensorflow-pull-requests-cpu-python3/7941/consoleFull

This one points to a nonexistent page:
https://ci.tensorflow.org/job/tensorflow-pull-requests-xla/4108/

These looks strange, but I don't see them on Jenkins:
https://source.cloud.google.com/results/invocations/c8affece-3a4b-4eae-af9b-2593c0ffcae1/targets

Triggering test again. 

@tensorflow-jenkins test this please

**Comment by drpngx** [comment]
If the test doesn't refresh and the link is broken then it's probably an old Jenkins target that was moved to kokoro.

**Comment by drpngx** [comment]
Some tests appear to fail. I'm not sure they are related. Could you check?

**Comment by freedomtan** [comment]
I don't think those failures are relevant. Anything I can do to clarify it?

**Comment by caisq** [comment]
@freedomtan The test failures are unrelated. Mergin PR. Thanks.

**Pull request merged**